lib/ansible/modules/async_wrapper.py

```diff
@@ -172,9 +170,8 @@ def _run_module(wrapped_cmd, jid):
                                   stderr=subprocess.PIPE)
 
         (outdata, stderr) = script.communicate()
-        if PY3:
-            outdata = outdata.decode('utf-8', 'surrogateescape')
-            stderr = stderr.decode('utf-8', 'surrogateescape')
+        outdata = outdata.decode('utf-8', 'surrogateescape')
+        stderr = stderr.decode('utf-8', 'surrogateescape')
```

Comment:
So instead of doing the conversions here, we can pass `text=True` (it's an alias of `universal_newlines=True` that appeared in Python 3.7) to the `Popen()` call.
```suggestion
```
WDYT?